PENS_v2.0.0_Draft.md

```diff
@@ -0,0 +1,1141 @@
+11 October 2016
+PENS v2.0.0 _**DRAFT**_
+
+DOCUMENT No. CMI010
+
+# Guidelines for Package Exchange Notification Services
+
+    VERSION 2.0.0
+	 _Proposed_ RELEASE DATE 2017-March-15  
+    Based on Revision 1.0a released 2006-March-15
+
+FORMATTING NOTES:
+
+This document is formatted using
+[Markdown](http://daringfireball.net/projects/markdown/). The document relies
+on features of GitHub Flavored Markdown ("GFM") as described at:
+<https://guides.github.com/features/mastering-markdown/#GitHub-flavored-markdown>.
+For the most accurate rendering and to avoid potential misinterpretations due
+to formatting discrepancies, please view the documentation on GitHub or use a
+rendering engine supporting GFM.
+
+THIS DOCUMENT IS CONTROLLED BY:
+
+    cmi5 Workgroup, as successor to AICC CMI Subcommittee
+
+    ALL REVISIONS SHALL BE APPROVED BY THE ABOVE ORGANIZATION PRIOR TO RELEASE.
+    
+POINT OF CONTACT:
+
+Tom King  
+cmi5 PENS Workgroup Lead  
+<tking@onpointlearning.com>
+ 
+FILED UNDER: <https://github.com/AICC/PENS_Spec_Current>
+
+---
+
+### Specification for Learning Technology—  Package Exchange Notification Services (PENS)
+
+#### Abstract:
+
+This specification describes a protocol to support a notification service to
+announce the location of content package(s) that are available for transport.
+The intent is to automate the notification, transfer and delivery confirmation
+of content packages between tools or systems that generate content and systems
+that manage, publish or deliver content. The scope of the specification is
+specifically constrained to the notification request, package transfer and
+related responses. Specifically outside the scope of this specification are
+mechanisms for physical deployment of content packages, content management,
+version control, publication or revocation of content.
+
+#### Keywords:
+CBT, CMI, cmi5, CMS, cmi5, content management, content package, e-learning,
+LCMS, LMS, notification service, package exchange, PENS, SCORM, xAPI.
+
+#### Revision Notes:
+The PEN 2.0.0 revision focuses on four goals:
+
+1. **Package format updates**: The addition of contemporary package formats
+such as xAPI and cmi5.
```

Comment:
@cawerkenthin depends what you mean by obsolete. 

If you mean that LMSs no longer need to support it, then there's a way to go. The major authoring tools would need to add cmi5 support (and drop the other packaging method), then all the content already published will need to get old and replaced. There's a few years to go yet!